firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalStoreTestCase.java

```diff
@@ -546,9 +548,10 @@ public void testHandlesPatchMutationThenDocumentThenAck() {
     int targetId = allocateQuery(query);
     applyRemoteEvent(
         addedRemoteEvent(
-            doc("foo/bar", 1, map("it", "base")).setHasLocalMutations(),
+            asList(doc("foo/bar", 1, map("it", "base")).setHasLocalMutations()),
```

Comment:
I think you can undo the change to "addedRemoteEvent". A remote doc should never have local mutations, so some of these tests are wrong.